processor/k8sattributesprocessor/documentation.md

```diff
@@ -4,33 +4,33 @@
 
 ## Resource Attributes
 
-| Name | Description | Values | Enabled |
-| ---- | ----------- | ------ | ------- |
+| Name | Description                                                                                                                                           | Values | Enabled |
```

Comment:
That file is automatically generated by the `metadata.yaml`. Please apply the edits at https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/3f9f88b8316345dd0cfc535ae1543e8412899cea/processor/k8sattributesprocessor/metadata.yaml#L109 and run `make generate` to update the `documentation.md` properly.

Reply:
done!